### ISSUE TITLE
Resume training is not working

### ISSUE THREAD

**Issue by jyothir07** [khornlund/severstal-steel-defect-detection#4]
I've successfully started training but when I tried resuming from the last epoch it is starting the training from epoch 0.

(steel) C:\Users\User\Documents\sdd>sever train --resume "C:\Users\User\Documents\sdd\saved\sever-FPN-efficientnet-b5-BCEDiceLoss-RAdam\1008_124332\checkpoints\checkpoint-epoch179.pth"
5740 - Runner - INFO - Using random seed: 447676
5740 - Runner - DEBUG - Building model architecture
Load result: None
5740 - Runner - DEBUG - Using device 0 of [0]
5740 - Runner - DEBUG - Building optimizer and lr scheduler
5740 - Runner - INFO - Found 308 encoder weight params
5740 - Runner - INFO - Found 193 encoder bias params
5740 - Runner - INFO - Found 19 decoder weight params
5740 - Runner - INFO - Found 12 decoder bias params
5740 - Runner - INFO - Loading checkpoint: C:\Users\User\Documents\sdd\saved\sever-FPN-efficientnet-b5-BCEDiceLoss-RAdam\1008_124332\checkpoints\checkpoint-epoch179.pth
5740 - Runner - INFO - Checkpoint "C:\Users\User\Documents\sdd\saved\sever-FPN-efficientnet-b5-BCEDiceLoss-RAdam\1008_124332\checkpoints\checkpoint-epoch179.pth" loaded
5740 - Runner - DEBUG - Getting augmentations
5740 - Runner - DEBUG - Getting data_loader instance
5740 - SamplerFactory - INFO - Creating `type`...
5740 - SamplerFactory - INFO - Sample population absolute class sizes: [5332 1801]
5740 - SamplerFactory - INFO - Sample population relative class sizes: [0.74751157 0.25248843]
5740 - SamplerFactory - INFO - Target batch class distribution [0.7846383 0.2153617] using alpha=-0.15
5740 - SamplerFactory - INFO - Rounded batch class distribution [0.75 0.25]
5740 - SamplerFactory - INFO - Expecting [6 2] samples of each class per batch, over 891 batches of size 8
5740 - SamplerFactory - INFO - Sampling rates: [1.00262566 0.98945031]
5740 - Runner - DEBUG - Getting loss and metric function handles
5740 - Runner - DEBUG - Initialising trainer
5740 - Trainer - INFO - Freezing encoder weights
5740 - Trainer - INFO - Starting training...
5740 - Trainer - INFO - Unfreezing encoder weights
C:\Users\User\Anaconda3\envs\steel\lib\site-packages\torch\nn\functional.py:1350: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
5740 - Trainer - DEBUG - Train Epoch: 0 [0/7128 (0%)] Loss: 0.049221
5740 - Trainer - DEBUG - Train Epoch: 0 [128/7128 (2%)] Loss: 0.051688

**Comment by khornlund** [comment]
Continuing from other epochs is not implemented
